tests/isotonic_test.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Isotonic Regression with PAV tests."""
+
+from absl.testing import absltest
+
+import jax
+import jax.numpy as jnp
+import numpy as onp
+
+from jax.test_util import check_grads
+from jaxopt._src.isotonic import isotonic_l2_pav
+from jaxopt._src import test_util
+
+
+class IsotonicPavTest(test_util.JaxoptTestCase):
+
+    def test_output_shape_and_dtype(self):
```

Comment:
add docstring to tests (could be a single phrase): https://g3doc.corp.google.com/company/readability/home/acquiring-readability/python-readability-advice.md?cl=head#test_docstrings

Reply:
Done